Assets/Scripts/SoundEffectManager.cs

```diff
@@ -0,0 +1,37 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using Util;
+
+public class SoundEffectManager : MonoBehaviour
```

Comment:
Ich würde das hier zu nem Singleton machen, die Methoden static machen und dann in Pizza und in Ingredient einfach über das Singleton laufen lassen. Dann musste nicht diese performance-lastigen Find()s benutzen :)

Reply:
Also dann einfach der Aufruf z.B. über `SoundEffectManager.GetRandomCompleteSoundEffect()`

Reply:
alles klar mach ich :)